server/config/auth/jwt.js

```diff
@@ -1,8 +1,11 @@
 import passport from 'passport';
 import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
```

Comment:
Let's pull out dotenv and Twilio things.

Reply:
✔️